restapi/src/main/java/io/stargate/web/docsapi/service/JsonConverterService.java

```diff
@@ -0,0 +1,290 @@
+package io.stargate.web.docsapi.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.*;
+import io.stargate.db.datastore.Row;
+import io.stargate.db.schema.Column;
+import io.stargate.web.docsapi.dao.DocumentDB;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
+public class JsonConverterService {
```

Comment:
`Nit:` Why not naming it as just `JsonConverter`? It's usually advisable not to overload a software's classes with suffixes like Manager, Service, Controller, etc.  

(please, feel free to ignore this comment 🙂 )

Reply:
Yeah I could rename - only reason why I named it this way is because DocumentService is named that way.

However, since the package name already states that these are service logic, I think it's reasonable to leave Service out